vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp

```diff
@@ -391,7 +416,9 @@ void ScrollViewShadowNode::UpdateZoomMode(const winrt::ScrollViewer &scrollViewe
                                                                    : winrt::ZoomMode::Disabled);
 }
 
-ScrollViewManager::ScrollViewManager(const Mso::React::IReactContext &context) : Super(context) {}
+ScrollViewManager::ScrollViewManager(const Mso::React::IReactContext &context) : Super(context) {
+  m_batchingEventEmitter = winrt::make_self<BatchingEventEmitter>(Mso::CntPtr(&context));
```

Comment:
>m_batchingEventEmitter [](start = 2, length = 22)

Can it be initialized in the initialization list instead? I.e. right after the Super above.